### ISSUE TITLE
Is it possible to change the name of the migrations table?

### ISSUE THREAD

**Issue by alessandroprudencio** [prisma/prisma#14806]
## Problem

by default generate the table __prisma_migrations_, but i have rules to follow in the database and one of them is that the name of the tables must be PascalCase. In typeorm this option already exists, is there any way to do it with the Prisma?

**Comment by janpio** [comment]
No, that name is currently hardcoded.

**Comment by alessandroprudencio** [comment]
> No, that name is currently hardcoded.

Ok, no problem, thanks for answering!

**Comment by janpio** [comment]
Let's keep this as a feature request - I think this makes total sense in situations like yours and we should consider this (sooner or later)👍

**Comment by kevinvdburgt** [comment]
Could this be a configuration in the `schema.prisma` file?

**Comment by janpio** [comment]
Yes, that would be the obvious way to configure things via a property of the `datasource`. 
Slight problem: We have been avoiding adding _any_ configuration if not absolutely necessary. We would need to change that and figure out what this exactly means.

**Comment by Kukunin** [comment]
Just to bring one more argument about this feature request. In our company architecture we have a bunch of micro-services that runs around a single big database. Multiple micro-services use Prisma and describe their own schema over the same database instance. Obviously, each micro-service tend to create it's own `_prisma_migrations` table and has conflicts there. Now we workaround this by having a separate schema per micro-service and having `?schema=microserviceA` in DATABASE_URL line. 

Having ability to define the name for the migrations table would simplify our life

**Comment by janpio** [comment]
@Kukunin Being able to change the migration table name would not actually be enough, as Prisma Migrate would try to delete the tables each microservice does not own. It is currently only fit to handle all tables in a schema it is connecting to. You will also want to 👍 https://github.com/prisma/prisma/issues/8864.

**Comment by Kukunin** [comment]
Actually, it's not a problem because for development/testing each micro-service lives in its own database. They are getting connected only on staging/production environments, and `prisma migrate deploy` is non-destructive, so it should be work, AFAIK

**Comment by janpio** [comment]
Interesting nuance, you are correct in that specific case.

**Comment by elvisdbl** [comment]
I also need this functionality to change the name of the _prisma_migrations table, how do I know when this change will be in the next versions of prisma?

**Comment by JayPe69** [comment]
Same need here

**Comment by carloscurottoba** [comment]
> Just to bring one more argument about this feature request. In our company architecture we have a bunch of micro-services that runs around a single big database. Multiple micro-services use Prisma and describe their own schema over the same database instance. Obviously, each micro-service tend to create it's own `_prisma_migrations` table and has conflicts there. Now we workaround this by having a separate schema per micro-service and having `?schema=microserviceA` in DATABASE_URL line.
> 
> Having ability to define the name for the migrations table would simplify our life

Were you able to tell prisma to put the migrations table inside the database shema? i see that it puts that table inside the default db schema, i wanted it to put it inside the specific schema specified in the db url

**Comment by sharathm** [comment]
> > Just to bring one more argument about this feature request. In our company architecture we have a bunch of micro-services that runs around a single big database. Multiple micro-services use Prisma and describe their own schema over the same database instance. Obviously, each micro-service tend to create it's own `_prisma_migrations` table and has conflicts there. Now we workaround this by having a separate schema per micro-service and having `?schema=microserviceA` in DATABASE_URL line.
> > Having ability to define the name for the migrations table would simplify our life
> 
> Were you able to tell prisma to put the migrations table inside the database shema? i see that it puts that table inside the default db schema, i wanted it to put it inside the specific schema specified in the db url

This was super useful!! Thank you.

Perhaps we should update the docs.  We had the exact same use case and this worked perfectly.
